framework/source/class/qx/bom/Cookie.js

```diff
@@ -62,7 +62,7 @@ qx.Bootstrap.define("qx.bom.Cookie",
         end = document.cookie.length;
       }
 
-      return decodeURI(document.cookie.substring(len, end));
+      return unescape(document.cookie.substring(len, end));
     },
```

Comment:
Nope. That is fixed by another PR that is no merged yet, so I think this PR must avoid that change, so this PR don't break anything with actual sate of master. :sweat_smile: